src/components/AddMoney/hooks/__tests__/useMantecaDepositPolling.test.tsx

```
@@ -0,0 +1,62 @@
+ * it mirrors GET /manteca/deposit/:id/status so the QR screen can advance.
+ */
+import { renderHook, waitFor } from '@testing-library/react'
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import type { ReactNode } from 'react'
+
+const mockGetDepositStatus = jest.fn()
+jest.mock('@/services/manteca', () => ({
+    mantecaApi: { getDepositStatus: mockGetDepositStatus },
+}))
+
+// eslint-disable-next-line import/first -- must come after jest.mock
```

ESLint

[error] Definition for rule 'import/first' was not found.

(import/first)